dev/tools/controllerbuilder/pkg/llm/gemini.go

```diff
@@ -30,19 +30,23 @@ import (
 // BuildGeminiClient builds a client for the Gemini API.
 func BuildGeminiClient(ctx context.Context) (Client, error) {
 	var opts []option.ClientOption
-
 	if s := os.Getenv("GEMINI_API_KEY"); s != "" {
 		opts = append(opts, option.WithAPIKey(s))
 	}
 
+	model := os.Getenv("GEMINI_MODEL")
+	if model == "" {
+		model = "gemini-2.0-pro-exp-02-05"
+		// model = "gemini-2.5-pro-exp-03-25"
```

Comment:
May I ask why not default to 2.5?

Reply:
I dont think we need this anymore. superseded by #4267